extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -253,54 +293,49 @@ public function doWikiForm() {
 					$info = $this->doUpdateTags( $wgRequest );
 					break;
 				case "findtags":
-					#we have 2 things that post from this page :(
-					if( $wgRequest->getVal('wpSearchTag') != null ) {
-						$this->mSearchTag = $wgRequest->getVal('wpSearchTag');
+					# we have 2 things that post from this page :(
+					if ( $wgRequest->getVal( 'wpSearchTag' ) != null ) {
+						$this->mSearchTag = $wgRequest->getVal( 'wpSearchTag' );
 						$info = $this->doSearchTags( $this->mSearchTag );
 					}
 
-						$this->mRemoveTag = $wgRequest->getVal('remove_tag');
-						$this->mRemoveTags = $wgRequest->getIntArray('remove_tag_id');
-					if( $this->mRemoveTag != null &&
+						$this->mRemoveTag = $wgRequest->getVal( 'remove_tag' );
+						$this->mRemoveTags = $wgRequest->getIntArray( 'remove_tag_id' );
+					if ( $this->mRemoveTag != null &&
 						$this->mRemoveTags != null ) {
 						$info = $this->doMultiRemoveTags();
 					}
 					break;
 				case "masstags":
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 15 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)